packages/components/src/slot-fill/slot.js

```diff
@@ -20,31 +19,45 @@ import {
 	isEmptyElement,
 } from '@wordpress/element';
 
-class Slot extends Component {
+/**
+ * Internal dependencies
+ */
+import { Consumer } from './provider';
+
+const Slot = ( props ) => (
```

Comment:
It seems like `Slot` should be at the end of the file

Reply:
180351cd8c67d5f16b64c13bfcc13814401d1aff